samples/react-native/src/Screens/ErrorsScreen.tsx

```diff
@@ -248,6 +248,7 @@ const ErrorsScreen = (_props: Props) => {
         ) : null}
         <View style={styles.mainViewBottomWhiteSpace} />
       </ScrollView>
+      <FeedbackButton navigation={_props.navigation} />
```

Comment:
i: this button should hide itself once feedback is sent.

Reply:
Thank you for the suggestion @lucas-zimerman 🙇 
If we hide the button how can be set visible again so that the user provide another feedback?

Reply:
The decision to show or hide the feedback button should be left to the end user. To enable this, users should be informed when feedback has been sent/ignored, allowing them to decide whether the feedback button should reappear.

Reply:
Good point @lucas-zimerman 👍 

I think this should be feasible with the [onFormClose](https://github.com/getsentry/sentry-react-native/pull/4364/files#diff-e95c4cea1ecbca9dd81fec4e312a2327ffe196c11c0fca2fbbe5ff6488f700f4R144) and [onFormSubmitted](https://github.com/getsentry/sentry-react-native/pull/4364/files#diff-e95c4cea1ecbca9dd81fec4e312a2327ffe196c11c0fca2fbbe5ff6488f700f4R161) callbacks.
Do you think we should extend the sample to showcase this (probably on a separate PR)?

> this button should hide itself once feedback is sent.

Probably I'm wrong on this but as a user I would expect the button to remain always visible given that it is not necessarily linked with a crash/error report. E.g. if the developer places the button on a secondary screen of the app the user would always find it there for a consistent experience.